apps/explorer/app/page.tsx

```diff
@@ -41,38 +42,51 @@ export default async function HomePage() {
         },
       })
     ),
+    to(explored.hostMetrics()),
+    to(explored.blockMetrics()),
   ])
 
   const [latestBlocks, latestBlocksError] = await getLatestBlocks()
   const latestHeight = latestBlocks ? latestBlocks[0].height : 0
 
-  if (metricsError || latestBlocksError || exchangeRatesError || hostsError) {
+  if (
+    latestBlocksError ||
+    exchangeRatesError ||
+    hostsError ||
+    hostMetricsError ||
+    blockMetricsError
+  ) {
     console.log(new Date().toISOString(), {
-      metricsError,
       latestBlocksError,
       exchangeRatesError,
       hostsError,
     })
   }
 
   console.log(new Date().toISOString(), {
-    metrics: humanBytes(Buffer.byteLength(JSON.stringify(metrics || ''))),
     latestBlocks: humanBytes(
       Buffer.byteLength(JSON.stringify(latestBlocks || ''))
     ),
     exchangeRates: humanBytes(
       Buffer.byteLength(JSON.stringify(exchangeRates || ''))
     ),
     hosts: humanBytes(Buffer.byteLength(JSON.stringify(hosts || ''))),
+    hostMetrics: humanBytes(
```

Comment:
This byte size logging code was from debugging a past issue that I thought might be due to the payload size. If you find it useful we can leave it, but feel free to remove it.

Reply:
Removed!